pkg/didcomm/packer/api.go

```diff
@@ -13,6 +13,22 @@ import (
 	"github.com/hyperledger/aries-framework-go/spi/storage"
 )
 
+const (
+	// EncodingTypeV2 is the default JWE `typ` protected header value as per:
+	// https://identity.foundation/didcomm-messaging/spec/#didcomm-encrypted-message
+	// and
+	//nolint:lll
+	// https://github.com/hyperledger/aries-rfcs/blob/master/features/0044-didcomm-file-and-mime-types/README.md#detecting-didcomm-versions
+	// for DIDComm compliance.
+	EncodingTypeV2 = "application/didcomm-encrypted+json"
```

Comment:
You might want to call this `EnvelopeEncodingTypeV2`

Reply:
ok..

Reply:
updated